### DISCUSSION TITLE
How long the GitHub Avatar cache will be there?

### DISCUSSION THREAD

**Discussion by berviantoleo** [forem/forem#20842] [Q&A] [answered]
How long the GitHub Avatar cache will be there?

![github avatar url](https://github.com/forem/forem/assets/15927349/8fe79679-3586-490a-99e3-4479c4d46037)

I changed the GitHub Avatar a long time ago, however, the embedded GitHub profile image still used the old photo.

Is there any mechanism to fetch the newest one?

For embedding, I use {% embed pull request URL %}

**Comment by maestromac** [comment, marked as answer]
There's no mechanism to fetch the newest one but we can manually make those changes for you. Please reach out to yo@dev.to

**Reply by berviantoleo** [reply]
I asked to yo@dev.to and still didn't help because they didn't know how to do that. It's weird because I believe the cache come from their sites and not from GitHub.
